src/twisted/internet/test/test_base.py

```diff
@@ -193,6 +194,7 @@ def test_str(self):
         C{str}, includes the unsigned id of the instance, as well as its state,
         the function to be called, and the function arguments.
         """
+        DelayedCall.debug = False
```

Comment:
Shouldn't this also be a call to `self.patch()`?

I guess I've never altered an already-patched value in a test before, but if the `patch` call in `setUp` goes away for some reason, then this line will be altering global state.